charts/argocd-apps/values-integration.yaml

```diff
@@ -19,7 +19,7 @@ applications:
     - name: web.dbMigrationEnabled
       value: "true"
     - name: common.env.redisUrl
-      value: "redis://redis.ecs.integration.govuk-internal.digital:6379"
+      value: "redis://shared-redis.eks.integration.govuk-internal.digital"
```

Comment:
FYI the port number is definitely optional according to [the redis URI scheme spec](https://www.iana.org/assignments/uri-schemes/prov/redis).